.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```

Comment:
What was this for?

Reply:
My VSCode did it automatically, I'll check it out, we can revert